snappi/pytest-examples/README.md

```diff
@@ -0,0 +1,153 @@
+## Document version  
+Make sure you have the latest version of this readme file by executing:
+```
+git pull  # in the otg-examples repository, origin: https://github.com/open-traffic-generator/otg-examples.git
+```
+## Description
+The goal of this repository is to help you setup, deploy and run test using Keysight Elastic Network Generator.
```

Comment:
This is a too broad statement. There are other examples of how to use KENG as well. What is the specific usecase this example solves? Is this examples with pytest?